pkg/component/plutono/plutono.go

```diff
@@ -331,21 +387,51 @@ datasources:
 	return datasource
 }
 
-func (p *plutono) getDashboards(ctx context.Context) (map[string]string, error) {
+func (p *plutono) getDashboardsConfigMap(ctx context.Context, suffix string) (*corev1.ConfigMap, error) {
 	var (
+		configMap            *corev1.ConfigMap
 		requiredDashboards   map[string]embed.FS
 		ignorePaths          = sets.Set[string]{}
 		dashboards           = map[string]string{}
-		extensionsDashboards = strings.Builder{}
+		extensionsDashboards = utils.MustNewRequirement(v1beta1constants.LabelExtensionConfiguration, selection.Equals, v1beta1constants.LabelMonitoring)
+		labelSelector        = client.MatchingLabelsSelector{Selector: labels.NewSelector().Add(extensionsDashboards)}
 	)
 
-	if p.values.ClusterType == component.ClusterTypeSeed {
+	configMap = &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "plutono-dashboards",
+			Namespace: p.namespace,
+			Labels:    getLabels(),
+		},
+	}
+
+	if suffix != "" {
+		configMap.Name = configMap.Name + "-" + suffix
+	}
+
+	if p.values.IsGardenCluster {
+		if suffix == "garden" {
+			requiredDashboards = map[string]embed.FS{gardenDashboardsPath: gardenDashboards, gardenAndShootDashboardsPath: gardenAndShootDashboards}
+
+			dashboard, err := p.getAdditionalDashboards(ctx, labelSelector, []string{v1beta1constants.PlutonoConfigMapOperatorDashboard})
```

Comment:
```suggestion
			additionalDashboards, err := p.getAdditionalDashboards(ctx, labelSelector, []string{v1beta1constants.PlutonoConfigMapOperatorDashboard})
```

Nit: use plural for the name of the variable.